src/main/resources/db/changelog/version-1.0/07.10.2023.part1.create-login-attempts-table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE login_attempts
+(
```

Comment:
i prefer to not use constraints on fields lvl

Reply:
Why?